javascript/javascript-quiz.md

```diff
@@ -1871,3 +1871,34 @@ console.log(cat.name);
 - [x] Wild
 - [ ] Tabby
 - [ ] Athena
+
+#### Q133. What will this code output to the log?
+
+```js
+var thing;
+let func = (str = 'no arg') => {console.log(str);}
+func(thing);
+func(null);
+```
+
+- [ ] null no arg
+- [ ] no arg no arg
+- [ ] null null
+- [ ] no arg null
+
+#### Q134. What will this code print to the console?
+
+```js
+const myFunc = () => {
+  const a = 2;
+  return () => console.log("a is " + a);
+};
+const a = 1;
+const test = myFunc();
+test();
+```
+
+- [ ] a is 1
+- [ ] a is undefined
+- [ ] It won't print anything.
+- [ ] a is 2
```

Comment:
The answer `a is 2`. Just execute the code in [Javascript Playground](https://playcode.io/)